src/Sarif.Viewer.VisualStudio/TelemetryProvider.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.ApplicationInsights;
+using Microsoft.CodeAnalysis.Sarif.Converters;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Sarif.Viewer
+{
+    /// <summary>
+    /// Wrapper class for Application Insights client.
+    /// </summary>
+    internal static class TelemetryProvider
```

Comment:
>TelemetryProvider [](start = 26, length = 17)

tests for this class and other?

you can defer this work. suggest opening issues for it. we do want to get the telemetry into the viewer release if possible